spyder/plugins/editor/confpage.py

```diff
@@ -219,110 +332,59 @@ def setup_page(self):
-
-        indent_tab_layout = QHBoxLayout()
-        indent_tab_layout.addLayout(indent_tab_grid_layout)
-        indent_tab_layout.addStretch(1)
-
         eol_on_save_layout = QHBoxLayout()
         eol_on_save_layout.addWidget(convert_eol_on_save_box)
         eol_on_save_layout.addWidget(convert_eol_on_save_combo)
 
-        # -- Vertical layout
-        automatic_group = QGroupBox(_("Automatic changes"))
-        automatic_layout = QVBoxLayout()
-        automatic_layout.addWidget(closepar_box)
-        automatic_layout.addWidget(autounindent_box)
-        automatic_layout.addWidget(add_colons_box)
-        automatic_layout.addWidget(close_quotes_box)
-        automatic_group.setLayout(automatic_layout)
-
-        whitespace_group = QGroupBox(_("Trailing whitespace"))
-        whitespace_layout = QVBoxLayout()
-        whitespace_layout.addWidget(self.removetrail_box)
-        whitespace_layout.addWidget(strip_mode_box)
-        whitespace_layout.addWidget(self.add_newline_box)
-        whitespace_layout.addWidget(self.remove_trail_newline_box)
-        whitespace_group.setLayout(whitespace_layout)
-
-        indentation_group = QGroupBox(_("Indentation"))
-        indentation_layout = QVBoxLayout()
-        indentation_layout.addLayout(indent_tab_layout)
-        indentation_layout.addWidget(ibackspace_box)
-        indentation_layout.addWidget(tab_mode_box)
-        indentation_group.setLayout(indentation_layout)
-
-        eol_group = QGroupBox(_("End-of-line characters"))
         eol_layout = QVBoxLayout()
-        eol_layout.addWidget(check_eol_box)
+        eol_layout.addWidget(fix_eol_box)
         eol_layout.addLayout(eol_on_save_layout)
         eol_group.setLayout(eol_layout)
 
         # ---- Advanced tab
-        # -- Templates
-        templates_group = QGroupBox(_('Template'))
-        template_btn = self.create_button(
+        # -- Template group
+        template_group = QGroupBox(_('Template'))
+        template_button = self.create_button(
             text=_("Edit new file template"),
             callback=self.plugin.edit_template,
-            set_modified_on_click=True
+            set_modified_on_click=True,
         )
 
-        templates_layout = QVBoxLayout()
-        templates_layout.addSpacing(3)
-        templates_layout.addWidget(template_btn)
-        templates_group.setLayout(templates_layout)
+        template_layout = QVBoxLayout()
+        template_layout.addSpacing(3)
+        template_layout.addWidget(template_button)
+        template_group.setLayout(template_layout)
 
-        # -- Autosave
+        # -- Autosave group
         autosave_group = QGroupBox(_('Autosave'))
         autosave_checkbox = newcb(
             _('Automatically save a backup copy of unsaved files'),
-            'autosave_enabled')
+            'autosave_enabled',
+            tip=_("If Spyder quits unexpectedly, "
+                  "it will offer to recover them on next launch"),
```

Comment:
```suggestion
            tip=_("If Spyder quits unexpectedly, it will offer to recover"
                  "them on next launch"),
```

Odd formatting.

Reply:
It's [SemBr](https://sembr.org/)